### ISSUE TITLE
Quarterback backup sets

### ISSUE THREAD

**Issue by glennlunder** [temisu/ancient#2]
Would you consider supporting Quarterback (amiga backup program) sets for this? Happy to supply test material if so.

**Comment by temisu** [comment]
Quarterback backup is generally speaking lovely "ancient" format worthy of supporting here.

And looking the documentation available, supporting it seems possible. but there are some if's.

- There are multiple (incompatible) versions of QuarterBack. I'd prefer to support all of them, if that is not possible at least the most popular of them. So having test files from several versions would be strongly desired.
- Preferably an example from a single archive what the expanded data should look like.

In short, I'm willing to support it as long as it benefits the wider community. If you have good examples, upload them somewhere and send me a link by email. Thanks

**Comment by polluks** [comment]
Quarterback source code http://aminet.net/package/disk/bakup/quarterback_src

**Comment by temisu** [comment]
I'm de-scoping all archival formats from the scope of this library and close this ticket - this would be another project if I ever get there
